src/main/java/core/lib/PasswordValidator.java

```diff
@@ -0,0 +1,30 @@
+package core.lib;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordValidator implements ConstraintValidator<FieldMatch, Object> {
```

Comment:
```suggestion
public class PasswordValidator implements ConstraintValidator<FieldMatch, UserRequestDto > {
```

Reply:
I don't think it's a good idea. Furthermore, we can use the same annotation for any field, but if you put there UserRequestDto then we wouldn't be able to do that anymore.